### ISSUE TITLE
Black screen on "Alone in the Dark" after Title screen

### ISSUE THREAD

**Issue by cnterr** [libretro/opera-libretro#181]
After the initial credits the game gets stuck on a black screen.
I'm using the latest release of both Retroarch and core, also trying different settings than the default ones but it doesn't change anything.
I'm using a Redump ISO.

**Comment by oblivioncth** [comment]
@cnterr Do you not experience the horrible audio sample looping issues that make the game (IMO) unplayable?

**Comment by CountCristoforo** [comment]
I was able to replicate and solve this issue. There are two European versions of the game both in French & English. There is ~ 50 MB difference between the two files. When using the one with less MB (non NTSC version), it will indeed cause a black screen after initial credits. If you change the bios to European it will solve this issue. Although the 3DO is region locked, I don't think all games are compatible with the American BIOS and vice versa.

**Comment by trapexit** [comment]
Correct. There could be bugs or hard coded to behave differently when noticing the difference in region.

**Comment by gameblabla** [comment]
Alone in the Dark (Europe) needs ROMSQI support : https://github.com/gameblabla/3doh_gcw0/commit/90180e06a20dcfab2f668311063288bf2f925afe . Otherwise it does a bunch of crazy stuff and overwrites memory it should not. I didn't need the timing hack for it.

**Comment by trapexit** [comment]
What problem are you referring to? Alone in the Dark (Europe) boots up and plays fine using the european roms. Your patch should not be necessary if using the proper rom since it would merely work as expected and return the proper flags.

**Comment by gameblabla** [comment]
You're right, it does, my bad. Though on opera it seems to be displayed with no black bar at bottom for PAL. (which could be easily fixed)

**Comment by trapexit** [comment]
You need to set the Region to PAL1. I don't yet do auto detection of titles -> region and rom requirements.

**Comment by trapexit** [comment]
The title is a NTSC title hence it runs at letterboxed NTSC within PAL resolution. Hence it will look full screen if you leave it in NTSC rendering mode.
